bins/stackabled/Cargo.toml

```diff
@@ -18,3 +18,5 @@ tracing = { workspace = true }
 snafu = { workspace = true }
 tokio = { workspace = true }
 stackabled-web = { path = "../../web", optional = true }
+serde.workspace = true
+k8s-openapi.workspace = true
```

Comment:
Maybe we should be consistent which format we use. But this is a nitpick.

```
serde = { workspace = true }
# VS
serde.workspace = true
```

Reply:
The latter is what `cargo add` adds if the dependency is already listed in `workspace.dependencies`, so I'd suggest standardizing on that.